lib/iolimiter/limitedreader.go

```diff
@@ -0,0 +1,33 @@
+// Copyright (C) 2014 The Syncthing Authors.
+//
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this file,
+// You can obtain one at http://mozilla.org/MPL/2.0/.
+
+package iolimiter
```

Comment:
If we don't use these from anywhere else right now, just having them in the connection package is fine.